docs/2_2_common_mechanisms.adoc

```diff
@@ -25,7 +25,7 @@ _[For an example of `NULL` being explicitly allowed see <<resourcePath>>._
 _Defensive implementations should still guard against `NULL` pointers.]_
 
 * The FMI Standard does not provide a runtime platform or portability layer.
-Access to operating system resources and services, such as memory and file system, must be implemented with special care because the availability of such resources and services is not guaranteed.
+Access to operating system resources and services, such as memory, network or file system, must be implemented with special care because the availability of such resources and services is not guaranteed.
```

Comment:
Very good. Maybe "... is not guaranteed on every simulator.".

Reply:
Yes, I like that addition. Done.